src/main/scala/seek/aws/sns/SubscribeTopic.scala

```diff
@@ -34,34 +38,69 @@ class SubscribeTopic extends AwsTask {
       _  <- subscribeToTopics(ts, ps).run(c)
     } yield ()
 
+
   private def subscribeToTopics(ts: List[String], ps: List[PendingSubscription]): Kleisli[IO, AmazonSNS, Unit] =
     Kleisli { c =>
       ts.foldLeft(IO.unit) { (z, t) =>
         for {
           _  <- z
           es <- listSubscriptions(t).run(c).compile.toList
-          _  <- addSubscriptions(t, uniquePending(ps, es).toList).run(c)
+          _  <- updateSubscriptions(t, merge(ps, es)).run(c)
         } yield ()
     }
   }
 
-  private def addSubscriptions(topicArn: String, ps: List[PendingSubscription]): Kleisli[IO, AmazonSNS, Unit] =
+  private def updateSubscriptions(topicArn: String, ps: List[PendingSubscription]): Kleisli[IO, AmazonSNS, Unit] =
+    ps match {
+      case Nil    => Kleisli.liftF(IO.unit)
+      case h :: t => for {
```

Comment:
For consistency with the rest of the codebase can you put multiline statements within a `for` comprehension on a newline after the arrow?